Source/IntegrationTests/TestFiles/LitTests/LitTest/git-issues/git-issue-4011.dfy.rs.check

```diff
@@ -0,0 +1 @@
+CHECK: error: expected identifier, found `<`
```

Comment:
Is this a bug in the Rust compiler that needs to be worked around for CI to pass? If so, we should prioritize fixing that.

Reply:
The Rust compiler is incomplete and already has a lot of these check files so we can still test it in CI. This gap is almost certainly already known and mentioned in other `.check` files. Even if it isn't we're already using the `.check` files to track these gaps.